package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "cho-cho",
+  "version": "1.0.0",
+  "description": "## Instructions",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Ainsley929/last-train-to-nodeville.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Ainsley929/last-train-to-nodeville/issues"
+  },
+  "homepage": "https://github.com/Ainsley929/last-train-to-nodeville#readme",
+  "devDependencies": {
+    "chai": "^4.2.0",
+    "index": "^0.4.0",
```

Comment:
Not sure what index is, betting that was a bad npm install at some point. Just be careful in projects when you make changes to your dependency lists. You want to always make sure you are only installing the things you need.